devito/builtins.py

```diff
@@ -92,6 +92,7 @@ def __exit__(self, exc_type, exc_value, traceback):
             self.v = comm.allreduce(np.asarray(self.n.data), self.op)[0]
 
 
+@dv.switchconfig(log_level='ERROR')
```

Comment:
Why?

Reply:
because ``norm`` produces the usual Operator log, which is painful (especially if you are in PERF/DEBUG mode) if you have codes with lots of Functions (eg elastic) for which a norm needs to be computed. With visco-elatic, the stdout is basically unreadable w/o this

Reply:
I get that, but hard coding it here is not great, would change the config in the code that uses it not here

Reply:
I don't think there's anything to be concerned about. It will still emit errors if there are errors; I don't think anybody cares about what the DSE/DLE do to the norm operator, and nobody will care if it takes 1 or 2 secs